pom.xml

```diff
@@ -112,7 +112,7 @@
   </dependencies>
 
   <build>
-    <finalName>csvMetaAdder</finalName>
+    <!--finalName>csvMetaAdder</finalName-->
```

Comment:
Why not remove vs. commenting out? Git will have a history of it if we want to return to it.